pkg/roachpb/BUILD.bazel

```diff
@@ -53,6 +53,52 @@ go_library(
     ],
 )
 
+# Creating bootstrap library as we need to partition
+# the dependencies here to avoid cyclical structures.
+go_library(
+    name = "bootstrap",
+    srcs = [
+        "api.go",
+        "app_stats.go",
+        "batch.go",
+        "batch_generated.go",
```

Comment:
Hm, this is broken. `batch_generated.go` is actually what we want to generate through `genbatch`. I see below we defined a separate `batch-gen.go` target. That's actually not what we want. What we're effectively doing in this PR is using the checked in generated file to generated the next (`batch-gen.go`), which defeats the whole purpose of defining the bootstrap target.

Reply:
Actually nevermind, we do have to do it this way -- just reminded myself why partitioning out pkg/roachpb into an independently compilable "bootstrap" target is painful. I guess we'll want to do what we did with langgen and soon introduce an out-of-band mechanism to implant generated code (batch-gen.go) back into the source tree (as batch_generated.go).